### ISSUE TITLE
CC0 License Request

### ISSUE THREAD

**Issue by RevoSucks** [n64decomp/mk64#474]
Hi!

You are mentioned here because you previously contributed to the n64decomp/mk64 repo, a decompilation of the game Mario Kart 64.

There is a move to license the source of that repo under CC0, and for that we need your permission, as past contributors.

**Why?**

The mk64 decomp source code is currently under no explicit license.

This means authors reserve all rights about the source being shared, modified, ... (see for example https://choosealicense.com/no-permission/) besides the basic rights granted by the GitHub TOS, mainly forking.

This makes many people feel uneasy about using the source, since it's not explicitly allowed to be used.

Note the unofficial stance though, is that people can do whatever they want with the code without having to ask or credit.

The goal is to make that stance official. However to do it properly we need to ask the authors if they are okay with it.

**Which license?**

CC0 is a license that seems to fit the unofficial stance.

It can be thought of as "public domain", or a more robust version of the Unlicense.

It also has the added benefit that it does not consist of a copyright affirmation, which is good given the murky waters decompilation is in.

**Read about CC0:**

-  https://creativecommons.org/publicdomain/zero/1.0/
-  https://choosealicense.com/licenses/cc0-1.0/
-  https://wiki.creativecommons.org/wiki/CC0_FAQ

If you don't care, the best choice to do for the project and users of the project (researchers, fans, modders...) is to agree.

If you agree, you can reply to this issue with some message like "I agree to share all my contributions to the mk64decomp under CC0"

You can also privately DM me on the [N64 Decompilation Discord](https://discord.gg/DuYH3Fh), or MegaMech on the same discord, if you do not wish to leave a public message here.

--------------------------------------------------

@Tmcg2 @MegaMech @CoderStig @ItzSwirlz @coco875 @ChiefFruitcake @AloXado320 @MaciekBaron @duncanspumpkin @farisawan-2000 @TommiRommi @tehzz @valandil @VDavid003 @iProgramMC @spazzylemons @i-do-cpp @someone2639

**Comment by VDavid003** [comment]
I agree to share all my contributions to the mk64decomp under CC0
(on an other note, I'm not sure who would even dare to try using this commercially, since CC0 permits that)

**Comment by ItzSwirlz** [comment]
I agree to share my contributions with the CC0 license!@

**Comment by Tmcg2** [comment]
I agree to share all my contributions to the mk64decomp under CC0

**Comment by AloXado320** [comment]
I agree to share all my contributions to the sm64 decomp under CC0

**Comment by AloXado320** [comment]
I agree to share all my contributions to the mk64 decomp under CC0.

**Comment by ChiefFruitcake** [comment]
I agree to share all my contributions to the mk64decomp under CC0

**Comment by coco875** [comment]
I agree to share all my contributions to the mk64decomp under CC0

**Comment by CoderStig** [comment]
I agree to share all my contributions to the mk64decomp under CC0.

Drive on.

**Comment by duncanspumpkin** [comment]
I agree to share all my contributions to the mk64decomp under CC0

**Comment by MaciekBaron** [comment]
I agree to share all my contributions to the mk64decomp under CC0

**Comment by TommiRommi** [comment]
I agree to share all my contributions to the mk64decomp under CC0

**Comment by spazzylemons** [comment]
I agree to share all my contributions to the mk64decomp under CC0

**Comment by valandil** [comment]
I agree to share all my contributions to the mk64decomp under CC0.

**Comment by someone2639** [comment]
I agree to share all my contributions to the n64decomp/mk64 repository under CC0

**Comment by farisawan-2000** [comment]
I agree to share all my contributions to the n64decomp/mk64 repository under CC0

**Comment by iProgramMC** [comment]
I agree to share my contribution to the n64decomp/mk64 repository under the CC0 license.
